enhancements/rhcos/static-networking-enhancements.md

```diff
@@ -0,0 +1,106 @@
+---
+title: improved-static-networking-configuration
+authors:
+  - "@miabbott"
+reviewers:
+  - "@ashcrow"
+  - "@dustymabe"
+  - "@dhellmann"
+  - "@cgwalters"
+approvers:
+  - "@ashcrow"
+  - "@cgwalters"
+  - "@crawford"
+  - "@imcleod"
+  - "@runcom"
+creation-date: 2020-04-22
+last-updated: 2020-05-12
+status: implementable
+see-also: https://github.com/openshift/enhancements/pull/210
+---
+
+# Improved Static Networking Configuration
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhancement describes improvements to how static networking is configured on Fedora CoreOS (FCOS) and RHEL CoreOS (RHCOS).  It covers improving the configuration of static networking for bare metal and VMWare hosts.
```

Comment:
I think the scope should be bigger, this seems focused on static networking but the use cases are much broader. Even if they have DHCP, they may still need VLAN's or bonding. There are also use cases to disable entire interfaces to speed up boot times.

See: https://github.com/coreos/ignition/issues/979

Reply:
Yep, I am pretty sure you're right.  Let's take this sub-thread to https://github.com/coreos/ignition/issues/979#issuecomment-632071056